molo/globalsite/migrations/0001_initial.py

```diff
@@ -35,6 +35,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('is_globalsite', models.BooleanField(default=False, help_text=b'It will set this site as the global site', verbose_name=b'Activate Golobal Site')),
                 ('autoredirect', models.BooleanField(default=False, help_text=b'When this is activated it will automatically redirect the users to the country of their choice.', verbose_name=b'Activate Auto Redirect')),
+                ('geolocation', models.BooleanField(default=False, help_text=b'When Geolocation is activated it will automatically redirect the users to the country site of their location.', verbose_name=b'Activate Geolocation')),
```

Comment:
Changing this in an existing migration would be a problem if there are ever sites that have deployed this migration before, but, seeing as this is new work as well, I assume there are no sites using this code yet, right?

Reply:
@jpaidoussi thanks, that is right, the plugin is new and haven't been released yet.